core/frontend/src/RealityDataSource.ts

```diff
@@ -5,15 +5,44 @@
 /** @packageDocumentation
  * @module Tiles
  */
-import { request, RequestOptions } from "./request/Request";
-import { AccessToken, BentleyStatus, GuidString, Logger } from "@itwin/core-bentley";
-import { IModelError, OrbitGtBlobProps, RealityData, RealityDataFormat, RealityDataProvider, RealityDataSourceKey, RealityDataSourceProps } from "@itwin/core-common";
+import { GuidString, Logger } from "@itwin/core-bentley";
+import { Cartographic, EcefLocationProps, OrbitGtBlobProps, RealityData, RealityDataFormat, RealityDataProvider, RealityDataSourceKey } from "@itwin/core-common";
 import { FrontendLoggerCategory } from "./FrontendLoggerCategory";
-import { IModelApp } from "./IModelApp";
-import { CesiumIonAssetProvider, ContextShareProvider, getCesiumAccessTokenAndEndpointUrl, getCesiumAssetUrl, getCesiumOSMBuildingsUrl } from "./tile/internal";
+import { CesiumIonAssetProvider, ContextShareProvider, getCesiumAssetUrl } from "./tile/internal";
+import { RealityDataSourceTilesetUrlImpl } from "./RealityDataSourceTilesetUrlImpl";
+import { RealityDataSourceContextShareImpl } from "./RealityDataSourceContextShareImpl";
+import { RealityDataSourceCesiumIonAssetImpl } from "./RealityDataSourceCesiumIonAssetImpl";
+import { Range3dProps } from "@itwin/core-geometry";
 
-/**
- * This interface provide methods used to access a reality data from a reality data provider
+const loggerCategory: string = FrontendLoggerCategory.RealityData;
+
+/** This interface provide spatial location and volume of interest, in meters, centered around `spatial location`
+ * @alpha
+ */
+export interface SpatialLocationAndExtents {
+  /** location of the point at the center of the reaity data */
+  location: Cartographic | EcefLocationProps;
+  /** extents of the volume around location */
+  worldRange: Range3dProps;
+  /** true if this reality data is geolocated */
+  isGeolocated: boolean;
```

Comment:
Under what circumstances would `isGeolocated` be false? Doesn't the existence of a location and worldRange imply it's geolocated?

Reply:
Some dataset that came from ContextCapture are in a custom coordinate system for which we cannot compute valid lat / long / elevation. Nothing in the dataset indicate that, thus when we found a Reality Data that is more than 5 Km below the ground, we mark it as not geolocated. For these dataset, we can still compute location and worldRange and we use these info to create a blank iModelConnection, so we can display them.

Reply:
See my edit above. Can you clarify documentation please?

Reply:
Push updated doc.